pkg/generator/helper/helper.go

```diff
@@ -278,6 +353,22 @@ func stepTestTemplate(myStepInfo stepInfo) []byte {
 	return generatedCode.Bytes()
 }
 
+func stepImplementation(myStepInfo stepInfo) []byte {
```

Comment:
extract to own PR

Reply:
A lot of generator changes are related to the enhancement ...